MainRepository/Program.cs

```diff
@@ -1,7 +1,26 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿// Program to convert the decimal number to binary and hexadecimal number.
+Console.Write ("Enter the number: ");
+List<int> binary = new ();
+List<char> hexadecimal = new ();

```

Comment:
Declare variable where it is used. binary should be after line 7 and hexadecimal should be after line 12.